istio.tf

```
@@ -27,6 +27,8 @@ resource "null_resource" "helm_repo" {
     curl -L https://git.io/getLatestIstio | ISTIO_VERSION=${var.istio_helm_release_version} sh -
```

Skip curl progress display
```suggestion
    curl -sL https://git.io/getLatestIstio | ISTIO_VERSION=${var.istio_helm_release_version} sh -
```